docs/how-to/service-dependencies.md

```diff
@@ -1,11 +1,297 @@
+```
+
+## Putting it together
+
+With the service dependencies and start order features, let's reconfigure our web app layer to:
+
+`001-my-web-app-layer.yaml`:
+
+```yaml
+services:
+  database:
+    override: replace
+    summary: database
+    command: python3 -m http.server 3306
+    startup: enabled
+  backend-server:
+    override: replace
+    summary: backend
+    command: python3 -m http.server 8081
+    startup: enabled
+    requires:
+      - database
+    after:
+      - database
+  frontend-server:
+    override: replace
+    summary: frontend
+    command: python3 -m http.server 8080
+    startup: enabled
+    requires:
+      - backend-server
+    after:
+      - backend-server
+```
+
+If we try to start it now:
+
+```{terminal}
+   :input: pebble run
+2024-06-28T02:17:23.347Z [pebble] Started daemon.
+[backend-server database frontend-server]
+2024-06-28T02:17:23.353Z [pebble] POST /v1/services 2.613042ms 202
+2024-06-28T02:17:23.356Z [pebble] Service "backend-server" starting: python3 -m http.server 8081
+2024-06-28T02:17:24.363Z [pebble] Service "database" starting: python3 -m http.server 3306
+2024-06-28T02:17:25.370Z [pebble] Service "frontend-server" starting: python3 -m http.server 8080
+2024-06-28T02:17:26.385Z [pebble] Started default services with change 1.
+```
+
+And we can verify the status of the services:
+
+```{terminal}
+   :input: pebble services
+Service          Startup  Current  Since
+backend-server   enabled  active   today at 10:17 CST
+database         enabled  active   today at 10:17 CST
+frontend-server  enabled  active   today at 10:17 CST
+```
+
+OK, there seems no difference when everything is working. But when one service fails to start, we can see the difference - for example, when the DB can't be started (say, the port is already taken):
+
+```{terminal}
+   :input: pebble run
+2024-06-28T02:28:06.569Z [pebble] Started daemon.
+[database backend-server frontend-server]
+2024-06-28T02:28:06.575Z [pebble] POST /v1/services 3.212375ms 202
+2024-06-28T02:28:06.578Z [pebble] Service "database" starting: python3 -m http.server 3306
+2024-06-28T02:28:06.627Z [pebble] Change 1 task (Start service "database") failed: cannot start service: exited quickly with code 1
+2024-06-28T02:28:06.633Z [pebble] GET /v1/changes/1/wait 57.610375ms 200
+2024-06-28T02:28:06.633Z [pebble] Started default services with change 1.
+```
+
+If we check the state of the services:
+
+```{terminal}
+   :input: pebble services
+Service          Startup  Current   Since
+backend-server   enabled  inactive  -
+database         enabled  inactive  -
+frontend-server  enabled  inactive  -
+```
+
+We can see that since the database failed to start, the backend and the frontend are not started. We can further verify this in Pebble [changes and tasks](../reference/changes-and-tasks/changes-and-tasks.md):
```

Comment:
```suggestion
We can see that since the database failed to start, the backend and the frontend are not started. We can further verify this using [changes and tasks](../reference/changes-and-tasks/changes-and-tasks.md):
```